internal/command/admin/user/user.go

```diff
@@ -45,13 +45,16 @@ func (u *CreateUser) Run(ctx context.Context, userHandler client.UserHandler, lo
 
 func (u *User) Help() string {
 	return `
-Create a new FunLess user. A user is a (name, token) pair, used to authenticate to the FunLess API.
+DESCRIPTION
+	
+	Create a new FunLess user. A user is a (name, token) pair, used to 
+	authenticate to the FunLess API.
+	To create new user specify an unique name. The token will be generated 
```

Comment:
To create *a* new user, it misses the article. It might be my own typo lol but just correct it with your next commit, thanks